curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c218c028c56a411b2a379a.md

```diff
@@ -11,17 +11,24 @@ To see your new HTML content for the `targetInputContainer`, you will need to us
 
 The `innerHTML` property sets or returns the HTML content inside an element. Here is an example of how to use it:
 
+Suppose you have the following form:
+
 ```html
-<div id="container">
-  <p>Example paragraph</p>
-</div>
+<form id="form">
+  <label>First name</label>
+  <input type="text">
+</form>
 ```
 
+Now say you want to add another label and input within the form. You can do so using the `+=` operator to add more HTML to the "inside" of the form.
```

Comment:
You need to connect it back to the `innerHTML` property that is the new concept they are learning about. 

```suggestion
If you want to add another `label` and `input` element inside the form, then you can use the `innerHTML` like this:
```

Reply:
How about the following statement instead to avoid using the word "this"?:

`If you want to add another `label` and `input` element inside the form, then you can use the `innerHTML` property as shown below:`

Reply:
Yeah, that works too 👍